bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/handler/EaseeThingHandler.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.easee.internal.handler;
+
+import static org.openhab.binding.easee.internal.EaseeBindingConstants.CHANNEL_TYPEPREFIX_RW;
+
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.easee.internal.Utils;
+import org.openhab.binding.easee.internal.command.EaseeCommand;
+import org.openhab.binding.easee.internal.config.EaseeConfiguration;
+import org.openhab.core.thing.Channel;
+import org.openhab.core.thing.ChannelGroupUID;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingUID;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * public interface of the {@link EaseeThingHandler}. provides some default implementations which can be used by both
+ * BridgeHandlers and ThingHandlers.
+ *
+ * @author Alexander Friese - initial contribution
+ */
+@NonNullByDefault
+public interface EaseeThingHandler extends ThingHandler, ChannelProvider {
+    static final Logger LOGGER = LoggerFactory.getLogger(EaseeThingHandler.class);
```

Comment:
To speed up PR review, I would suggest to use an instance variable for the logger to avoid the "Logger should be non-static field (SLF4J)" SAT finding. I have asked other maintainers for advice, because the check can either be valid and there for a good reason, or it could be an old left-over. The safest approach therefore would be to follow the advice, which wouldn't cause any significant harm.